controllers/provider-azure/pkg/internal/imagevector/imagevector.go

```diff
@@ -39,7 +39,7 @@ func init() {
 	runtime.Must(err)
 }
 
-// ImageVector is the image vector that contains all the needed images.
+// ImageVector is the image vector that contains al the needed images
```

Comment:
```suggestion
// ImageVector is the image vector that contains all the needed images.
```

Reply:
ok, and I will also do this for the other providers if it's not since I opened my pr